src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/PropertyPageDataProducer.cs

```diff
@@ -124,6 +124,18 @@ IEnumerable<IEntityValue> createPropertyPageValuesAsync()
                     if (projectCatalog.GetSchema(schemaName) is Rule rule
                         && !rule.PropertyPagesHidden)
                     {
+                        if (rule.Name == "RazorGeneral" || rule.Name == "RazorExtension")
```

Comment:
Should this use StringComparers.RuleNames?

Reply:
Instead of a separate if-statement, it might make sense to have a static list of ignored rule names and have the first if-statement above this one do an `&& !ignoredRuleNames.Contains(rule.Name)` or something of the sort. Then, we're just filtering them out as part of the already existing workflow.